### ISSUE TITLE
Tidy Code does not indent nested blocks properly if braces are missing

### ISSUE THREAD

**Issue by Hari3** [processing/p5.js-web-editor#1815]
#### Nature of issue? Bug

- Found a bug

#### Details about the bug: 

- Web browser and version: Chrome 89.0.4389.82
- Operating System: Windows 10
- Steps to reproduce this bug:
 1) Create a nested block(branch or loop), inside another branch or loop without using braces anywhere
   a)Check this [Sample editor file](https://editor.p5js.org/HariKrishnan/sketches/0wsJ6ZcvZ)
Here are the contents for ease of access
```
// Indent Issue
// Create a copy and Tidy Code (Edit -> Tidy Code)
// This is already indented properly
do
  if(true)
    console.log("foo")
  else
    console.log("bar")
while(false)
  
// Same is true for while and for loops:

for(i=0;i<10;i++)
  if(true)
    console.log("foo")
  else
    console.log("bar")

while(i<12)
  if(true)
    i++
  else
    i--
  
console.log("foo")

// Note that out put is same in before and after tiding code, "foo" is printed 12 times
```
2. Click on Edit > Tidy Code
  a. Expected, this code is already indented properly: if and else lines up. No change should occur
  b. Actual: else gets aligned with previous block(while, do or for)

**Comment by satyasaibhushan** [comment]
Hey @Hari3,
This will be resolved by the PR #1788. And will be reflected in the website whenever the next release happens.
So, no worries :)

**Comment by catarak** [comment]
Thanks for reporting! I just checked if this will indent in your expected way with the merge of #1788, and it still puts them on the same line. 

I think this is actually a matter of code style, and both prettier and beautify-js enforce that if/else statements without curly braces should be on one line. I personally agree with this rule and think it should be enforced—I rarely use if/else statements without curly braces because I think it can lead to bugs if you need to update the logic in the future and want to change the body of the if/else statements to have multiple lines.

I found some discussion in the prettier repository about this that seems to be unresolved: https://github.com/prettier/prettier/issues/3049
